chaoscenter/graphql/server/pkg/chaos_experiment/handler/handler.go

```diff
@@ -1548,6 +1573,8 @@ func (c *ChaosExperimentHandler) RunChaosWorkFlow(ctx context.Context, projectID
 			Completed:       false,
 			ResiliencyScore: &resScore,
 			ExecutionData:   string(parsedData),
+			Probes:          probes,
+			//TODO add probes
```

Comment:
Stale comment? I see probe is already there. If it's something else needs a better descriptive comment